modules/appeals_api/app/services/appeals_api/remove_pii.rb

```diff
@@ -15,13 +17,20 @@ def initialize(form_type:)
     end
 
     def run!
-      validate_form_type!
+      Datadog::Tracing.trace("#{self.class.name} - #{form_type.name}") do
```

Comment:
I used ddtrace here because the only logs we currently have for this job in Datadog are the `start` and `done` or `fail` messages - no details about the job.

I'll need to sync all these changes to lighthouse-vets-api after this ticket is complete. In that repo, datadog traces are enabled for all code, but this code, so I'll remove the `Datadog::Tracing.trace` wrapper when I copy this over.